pkg/graph/schema.resolvers.go

```diff
@@ -702,20 +702,21 @@ func (r *mutationResolver) UpdateSystemIntakeLinkedCedarSystem(ctx context.Conte
 
 // SetSystemIntakeGRBPresentationLinks is the resolver for the setSystemIntakeGRBPresentationLinks field.
 func (r *mutationResolver) SetSystemIntakeGRBPresentationLinks(ctx context.Context, input models.SystemIntakeGRBPresentationLinksInput) (*models.SystemIntakeGRBPresentationLinks, error) {
```

Comment:
https://github.com/CMS-Enterprise/easi-app/blob/6fc154d47380c564f670827196fb9de6d7f9e782/pkg/graph/resolvers/trb_request_document.go#L50-L86

Reply:
note, this is now handled in teh handleFile function with the resolvers.